src/dotnet/Azure.ClientSdk.Analyzers/ci.yml

```diff
@@ -0,0 +1,27 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
This can be a follow-up but we should probably move things from src/dotnet to tools directory.